### PR TITLE
Correct Section 1.13 Simple

### PR DESCRIPTION
**Pull request by abeeisnotabug**
#### Summary
Correct the transpose operations in the "Optimization through Cholesky Factorization" section. Please see advanced PR for more concise rework.

### CODE REVIEW

File: src/stan-users-guide/regression.Rmd [hunk 1]
```diff
@@ -1479,7 +1479,7 @@ factor of the final covariance matrix,
 ```
 Sigma_beta
   = quad_form_diag(Omega, tau)
-  = diag_post_multiply(L_Omega,tau) * diag_post_multiply(L_Omega,tau)'
+  = diag_post_multiply(L_Omega', tau)' * diag_post_multiply(L_Omega', tau)
```
Comment (bbbales2) [inline src/stan-users-guide/regression.Rmd:1482]: `diag_pre_multiply(tau, L_Omega) * diag_post_multiply(L_Omega', tau)`

**Review by bbbales2 — CHANGES_REQUESTED** [review]
One simplification, thanks for finding this!

**Comment by bbbales2** [comment]
@abeeisnotabug we can close this now right?

**Comment by abeeisnotabug** [comment]
> @abeeisnotabug we can close this now right?

Yes! Thank you engaging with all this!

**Pull request closed without merging**